block-explorer/elasticsearch/mappings/transaction.json

```diff
@@ -0,0 +1,59 @@
+{
+  "mappings": {
+    "dynamic": false,
+    "properties": {
```

Comment:
I am not sure if we still need `timestamp` field, cause probably we will use ordinal + address for search_after. What do you think?

Reply:
I would leave it for now